helper/utilty.py

```diff
@@ -15,7 +15,7 @@
 import tensorflow as tf
 from PIL import Image
 from os.path import isfile, join
-from scipy import misc
+import imageio as misc
```

Comment:
Please use scipy package for image writing. As far as I tested, image.imwrite() automatically rescale the image intensity and it brakes evaluation/inference. We need to use spicy without rescaling.